### ISSUE TITLE
Display ribbon on the product boxes over the picture when some discount is applied, "old price" is specified, marked as new (available as a plugin)

### ISSUE THREAD

**Issue by mariannk** [nopSolutions/nopCommerce#19]
This is how I did it for old price:
1. CSS: 

```
.product-grid .product-item .picture, .product-list .product-item .picture{position: relative;} 
.old-price-ribbon {-webkit-transform:rotate(-45deg);-moz-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-o-transform:rotate(-45deg);transform:rotate(-45deg);border:25px solid transparent;border-bottom: 25px solid #757575;position:absolute;top:0;left:-42px;padding:0 10px;width:65px;color:white;} 
.old-price-ribbon .txt{position:absolute;top:5px;left:25px;}
```
1. ProductBox.cshtml 

```
@if (!String.IsNullOrEmpty(Model.ProductPrice.OldPrice))
{
    <div class="old-price-ribbon">
        <div class="txt">
            @T("Promo.Sale")
        </div>
    </div>
}
```

right after </a> closing tag inside picture

Some other points:
1. Make this option configurable (move to UI)
2. Test it in IE8 (hide if not supported b display:none)
3. DO not forget about RTL theme
4. We can do the same for discounts. Contribution here - https://nopcommerce.codeplex.com/SourceControl/network/forks/nicolasmuniere/contibutions/contribution/3900
5. Also see existing plugins http://www.nopcommerce.com/extensions-and-themes.aspx?searchterm=ribbon
6.Implement for products marked as "NEW"

Let's discuss what exactly we should display (ribbons)

**Comment by tigran-avdalyan** [comment]
Ability to add custom ribbon for each product, for example Verified by Us, Guaranteed by Us , Or Vendor Logo

**Comment by AndreiMaz** [comment]
After some consideration decided not to implement out of the box  - already available as plugin

**Comment by tigran-avdalyan** [comment]
@AndreiMaz 
Hi Andrei, Yes available as plugin, but not for all versions.
NopCommerce 4.00 not support

**Comment by poyker** [comment]
@tigran-avdalyan 
Our (Nop-Templates) [Ribbons ](https://www.nopcommerce.com/p/1054/nop-product-ribbons-nop-templatescom.aspx)plugin is available for nopCommerce 4.00.
